AppWidget/app/src/main/java/com/example/android/appwidget/glance/GlanceTheme.kt

```diff
@@ -38,8 +38,9 @@ object GlanceTheme {
 internal val LocalColorProviders = staticCompositionLocalOf { dynamicThemeColorProviders() }
 
 /**
- * Temporary implementation of Material3 theme for Glance. It still requires to manually set the
- * colors to all Glance components
+ * Temporary implementation of Material3 theme for Glance.
+ *
+ * Note: It still requires to manually set the colors to all Glance components
```

Comment:
Suggest: This still requires manually setting the colors for all Glance components.